contentcuration/contentcuration/viewsets/websockets/signals.py

```diff
@@ -0,0 +1,67 @@
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from contentcuration.models import Change
+
+
+@receiver(post_save, sender=Change, weak=False)
+def broadcast_new_change_model_handler(sender, instance, created, **kwargs):
+    broadcast_new_change_model(instance)
+
+
+def broadcast_new_change_model(instance):
+    channel_layer = get_channel_layer()
+    serialized_change_object = Change.serialize(instance)
+    # Name of channel group
+    room_group_name = instance.channel_id
+
+    # name of indiviual_user group
+    indiviual_room_group_name = instance.user_id
+    print("DEBUG", instance.errored)
+    print("DEBUG", instance.__dict__)
+    # if the change object is errored then we broadcast the info back to indiviual user
+    if instance.errored:
+        async_to_sync(channel_layer.group_send)(
+            str(instance.created_by_id or ""),
```

Comment:
Since we really should have the `created_by_id` defined, perhaps we log an error if we detect it isn't set, and remove this passive `or ""` which could lead to undefined behavior.